src/plugins/hathor_pubsub.js

```diff
@@ -0,0 +1,78 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+/* istanbul ignore next */
+async function checkDeps() {
+  const requiredDeps = {
+    '@google-cloud/pubsub': '^4.0.1',
```

Comment:
I think having this dependency and not adding this package in the Docker image we send to Docker Hub is slightly odd.

Someone trying to use the plugin would be unable to, unless a custom image is built from ours that includes the needed package.

On the other hand, I'm not sure we want to always add the package just because of a plugin that a small percentage of the wallet-headless users will need.

I'm not sure the best way to deal with this. Maybe building a second Docker image that contains the package?

What do you think, @r4mmer @pedroferreira1 ?